.github/workflows/main.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         with:
           helmRelease: releases/stg/
           helmVersion: v3
+        continue-on-error: true
```

Comment:
Note, wasn't 100% sure if it would be best just to delete this whole stage (as the `stg` directory no longer exists), or to just ignore the error. I went with the later thinking that it'd probably be easier to re-enable it again later (although the risk is that it gets forgotten about). Maybe it'd just be best to remove the stage, and the namespace also...

Reply:
Happy to remove the entire stage.